models/netsuite2/netsuite2__balance_sheet.sql

```diff
@@ -143,11 +139,10 @@ balance_sheet as (
     null as account_id,
     null as account_number,
 
-    --The below script allows for accounts table pass through columns.
     {% if var('accounts_pass_through_columns') %}
-
-    null as {{ var('accounts_pass_through_columns') | join (", null as ")}} ,
-
+      {% for field in var('accounts_pass_through_columns') %}
+        null as {{ field.alias if field.alias else field.name }},
+      {% endfor %}
```

Comment:
Do you think this is something that would be common and should include in the macro? If not, we can keep this as is.

Reply:
yeah as we discussed in standup, i don't see us using this particular logic in more packages (especially given how complicated the macro would become)